src/test/end-to-end-tests/test-runner-bot.ts

```diff
@@ -203,7 +209,30 @@ async function mainLoop(): Promise<void> {
     }
 }
 
+async function pollVoiceConnectionReady(): Promise<void> {
+    const voiceChannel = getVoiceChannel();
+
+    // wait up to 2 minutes for vc to be ready
```

Comment:
Should it error out if the vc is still not ready 2 minutes in

Reply:
yes